### PR TITLE
Fix some grammar in docs [ci skip]

### PR DESCRIPTION
**Pull request by kenta-s**

### CODE REVIEW

File: actionpack/lib/action_dispatch/system_test_case.rb [hunk 1]
```diff
@@ -20,7 +20,7 @@ class SystemTestCase < IntegrationTest
     # <tt>application_system_test_case.rb</tt> file that is generated with a new
     # application or scaffold.
     #
-    # Here is an example system test:
+    # Here is an example of system test:
```
Comment (Erol) [inline actionpack/lib/action_dispatch/system_test_case.rb]: I don't know but the current one already looks correct to me.

File: actionpack/lib/action_dispatch/system_test_case.rb [hunk 2]
```diff
@@ -36,7 +36,7 @@ class SystemTestCase < IntegrationTest
     #     end
     #   end
     #
-    # When generating an application or scaffold a +application_system_test_case.rb+
+    # When generating an application or scaffold an +application_system_test_case.rb+
```
Comment (vipulnsward) [inline actionpack/lib/action_dispatch/system_test_case.rb]: comma after scaffold

File: actionpack/lib/action_dispatch/system_test_case.rb (hunk 1 above)
Comment (vipulnsward) [inline actionpack/lib/action_dispatch/system_test_case.rb]: The current one is ok. If this needs to change it should be- "example of a system test".

**Review by vipulnsward — CHANGES_REQUESTED** [review]

**Comment by kenta-s** [comment]
Thank you for pointing out my mistakes. they are fixed.

**Review by vipulnsward — APPROVED** [review]

**Pull request merged**